bindings/rascal/models/krr.py

```diff
@@ -1,9 +1,14 @@
-from ..utils import BaseIO
+from ..utils import BaseIO, is_notebook
 from ..lib import compute_sparse_kernel_gradients, compute_sparse_kernel_neg_stress
 
 import numpy as np
 import ase
 
+if is_notebook():
```

Comment:
Just a note, this includes `tqdm` as a strict dependency (as in, the file cannot be imported without it). Is this something we want to do? I'd rather make it optional, to be honest.